### PR TITLE
Set default value for cron_file

### PR DESCRIPTION
**Pull request by aleksandrzhiliaev**

### CODE REVIEW

**Comment by vgarvardt** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -154,8 +154,15 @@ deployment_post                     : False
                        #   delay_sec   : 5
                        #   timeout_sec : 60
 
-## Cron jobs TODO: Make an example of cron jobs
-deployment_cron_jobs                : "none"
+## Cron jobs
+deployment_cron_jobs:
+  - name              : "cron-job-name"
+    minute            : "0"
```
Comment (vgarvardt) [inline README.md]: add comments for required/optional fields - it will be very useful

**Comment by vgarvardt** [comment]
👍

**Comment by c4pone** [comment]
+1

**Pull request merged**